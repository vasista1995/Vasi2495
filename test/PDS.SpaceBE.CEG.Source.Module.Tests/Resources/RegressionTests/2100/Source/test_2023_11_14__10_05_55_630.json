{
  "Plant": "0600",
  "IdSystemName": "HPSModFAUF",
  "Line": "HD",
  "ProductionOrder": "81773909",
  "MaterialNumber": "564",
  "MeasurementRecipe": "Abart 14 Hybrid Pack Drive",
  "ProcessEquipment": "10123714",
  "OrderType": "2100",
  "QrkGroup": "2100",
  "ProcessGroup": "Frame_Assembly",
  "SpcClass": "2100",
  "ParameterType": "2100",
  "LongParameterName": "Klebemenge",
  "MaterialNumberText": "2100",
  "ProcessEquipmentName": "ZBV5 -RM",
  "Operator": "2100",
  "ParameterClass": "NON_PPAP",
  "Segment": "HPS",
  "Package": "HD",
  "GroupId": "glue_weight",
  "Module": "13 assembly",
  "PosName": "2100",
  "FourDReport": "*",
  "VariableExtr1": "564",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 332712759,
      "Seqnr": 1,
      "Value": 0.506,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "UniqueSourceKey": "CEG:BE:81773909:HD:gluing:glue_weight:105699:332712759:0",
  "PKey": "CEG:BE:81773909:HD:gluing:glue_weight:105699:332712759:0",
  "ParameterName": "glue_weight",
  "ParameterUnit": "[g]",
  "SampleDate": "2023-10-27T11:34:43Z",
  "SampleDateUtc": "2023-10-27T09:34:43Z",
  "SampleSize": 1,
  "Sigma": "0",
  "Mean": ",506",
  "Median": ",506",
  "Min": ",506",
  "Max": ",506",
  "Q1": ",506",
  "Q3": ",506",
  "LdsID": "2100",
  "IsCalcParam": 0,
  "MeanCntrlLow": 0.469302041546872,
  "MeanCntrlTarget": 0.49829531914894,
  "MeanCntrlHigh": 0.527288596751008,
  "CntrlLow": 0.469302041546872,
  "CntrlTarget": 0.49829531914894,
  "CntrlHigh": 0.527288596751008,
  "RangeCntrlLow": 0.0,
  "RangeCntrlTarget": 0.0,
  "RangeCntrlHigh": 0.0,
  "SigmaCntrlLow": 0.0,
  "SigmaCntrlTarget": 0.0,
  "SigmaCntrlHigh": 0.0,
  "SpecLow": 0.45,
  "SpecTarget": 0.5,
  "SpecHigh": 0.55,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "Y",
  "CtrlHighEnabled": "Y",
  "CtrlRangeLowEnabled": "Y",
  "CtrlRangeHighEnabled": "Y",
  "CtrlSigmaLowEnabled": "Y",
  "CtrlSigmaCenterEnabled": "Y",
  "CtrlRangeCenterEnabled": "Y",
  "CtrlCenterEnabled": "Y",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "Y",
  "CreatedDate": "2023-10-27T11:34:43Z",
  "CreatedDateUtc": "2023-10-27T09:34:43Z",
  "UpdatedDate": "2023-10-27T11:34:43Z",
  "UpdatedDateUtc": "2023-10-27T09:34:43Z",
  "RvStoreFlag": "Y",
  "Operation": "gluing",
  "ChannelId": "105699",
  "CkcId": "0",
  "ChannelName": "glue_weight",
  "SpaceInstanceName": "BE",
  "Sitekey": "CEG",
  "CalcCreated": "02/01/2023 04:12:49",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 0.506571428571429,
  "ExtEwmaMV": 0.506245700400643,
  "ExtMsMV": 0.0,
  "OutOfOrder": "N",
  "EwmaS": 0.0,
  "EwmaR": 0.0,
  "Calc2ID": 9699168,
  "LimitEnable": "YYYYYNYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY",
  "ExtEwmaMVLCL": 0.483797763027914,
  "ExtEwmaMVCenter": 0.49829531914894,
  "ExtEwmaMVUCL": 0.512792875269966,
  "ExtMaMVLCL": 0.48733619683053,
  "ExtMaMVCenter": 0.49829531914894,
  "ExtMaMVUCL": 0.50925444146735,
  "EwmaSLCL": 0.0,
  "EwmaSCenter": 0.0,
  "EwmaSUCL": 0.0,
  "EwmaRLCL": 0.0,
  "EwmaRCenter": 0.0,
  "EwmaRUCL": 0.0,
  "ExtMSMVLCL": 1.43330794048562E-05,
  "ExtMSMVCenter": 0.00675904992175591,
  "ExtMSMVUCL": 0.0271513652914233,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 0.454805402745839,
  "RawUal": 0.541785235552042,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChannelType": "N",
  "ChState": "Online"
}