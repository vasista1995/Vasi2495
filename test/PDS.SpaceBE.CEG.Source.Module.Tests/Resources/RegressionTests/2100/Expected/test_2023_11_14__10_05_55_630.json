{
  "IdSource": "SPACEACT3:CEG:BE:81773909:HD:gluing:glue_weight:105699:332712759:0",
  "Data_flat_Metadata": {
    "Plant": "0600",
    "Line": "HD",
    "ProductionOrder": "81773909",
    "MaterialNumber": "564",
    "MeasurementRecipe": "Abart 14 Hybrid Pack Drive",
    "ProcessEquipment": "10123714",
    "OrderType": "2100",
    "QrkGroup": "2100",
    "ProcessGroup": "Frame_Assembly",
    "SpcClass": "2100",
    "ParameterType": "2100",
    "LongParameterName": "Klebemenge",
    "MaterialNumberText": "2100",
    "ProcessEquipmentName": "ZBV5 -RM",
    "Operator": "2100",
    "ParameterClass": "NON_PPAP",
    "Segment": "HPS",
    "Package": "HD",
    "GroupId": "glue_weight",
    "Module": "13 assembly",
    "PosName": "2100",
    "FourDReport": "*",
    "ParameterName": "glue_weight",
    "ParameterUnit": "[g]",
    "ParameterOper": "gluing",
    "SiteKey": "CEG",
    "ChannelId": "105699",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-27T09:34:43Z",
    "CreatedTimestamp": "2023-10-27T11:34:43Z",
    "UpdatedTimestamp": "2023-10-27T11:34:43Z",
    "UpdatedTimestampUtc": "2023-10-27T09:34:43Z",
    "ExportedTimestampUtc": "2023-10-27T11:34:43Z",
    "ExportedTimestamp": "2023-10-27T11:34:43Z",
    "SampleTimestampUtc": "2023-10-27T09:34:43Z",
    "SampleTimestamp": "2023-10-27T11:34:43Z",
    "SourceDataLevel": "C",
    "ChannelName": "glue_weight",
    "RvStoreFlag": "Y",
    "CalcCreated": "02/01/2023 04:12:49",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 9699168,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2100",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 0.55,
        "SpecLow": 0.45,
        "SpecTarget": 0.5,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYYYYNYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY"
      },
      "SpaceAggregates": {
        "Mean": ",506",
        "Median": ",506",
        "Sigma": "0",
        "Min": ",506",
        "Max": ",506",
        "Q1": ",506",
        "Q3": ",506"
      },
      "ControlLimits": {
        "ctrl_mean_low": 0.469302041546872,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 0.49829531914894,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 0.527288596751008,
        "CtrlMeanHighEnabled": "Y",
        "ctrl_low": 0.469302041546872,
        "CtrlLowEnabled": "Y",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "Y",
        "ctrl_target": 0.49829531914894,
        "ctrl_high": 0.527288596751008,
        "CtrlHighEnabled": "Y",
        "ctrl_sigma_low": 0.0,
        "CtrlSigmaLowEnabled": "Y",
        "ctrl_sigma_target": 0.0,
        "ctrl_sigma_high": 0.0,
        "CtrlSigmaHighEnabled": "Y",
        "ctrl_range_low": 0.0,
        "CtrlRangeLowEnabled": "Y",
        "ctrl_range_target": 0.0,
        "ctrl_range_high": 0.0,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 0.506571428571429,
        "ExtEwmaMV": 0.506245700400643,
        "ExtMsMV": 0.0,
        "EwmaS": 0.0,
        "EwmaR": 0.0,
        "ExtEwmaMVLCL": 0.483797763027914,
        "ExtEwmaMVCenter": 0.49829531914894,
        "ExtEwmaMVUCL": 0.512792875269966,
        "ExtMaMVLCL": 0.48733619683053,
        "ExtMaMVCenter": 0.49829531914894,
        "ExtMaMVUCL": 0.50925444146735,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 1.43330794048562E-05,
        "ExtMSMVCenter": 0.00675904992175591,
        "ExtMSMVUCL": 0.0271513652914233,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.454805402745839,
        "RawUal": 0.541785235552042,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 332712759,
        "Seqnr": 1,
        "Value": 0.506,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 1
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CEG:BE:HD::glue_weight:105699",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "HPSModFAUF:81773909",
    "IdSystemName": "HPSModFAUF",
    "Type": "Fauf"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CEG:BE:81773909:HD:gluing:glue_weight:105699:332712759:0"
  }
}